docs/concepts/workloads/vm.md

```diff
@@ -84,15 +84,45 @@ It is possible to update parts of an existing `VirtualMachine` resource. Some fi
+
+For an example, consider the following VM Class:
+```
+apiVersion: vmoperator.vmware.com/v1alpha1
+kind: VirtualMachineClass
+metadata:
+  name: my-vm-class
+spec:
+  configSpec:
+    _typeName: VirtualMachineConfigSpec
+    cpuAllocation:
+      _typeName: ResourceAllocationInfo
+      limit: 400
+      reservation: 200
+    memoryAllocation:
+      _typeName: ResourceAllocationInfo
+      limit: 2048
+      reservation: 1024
+  hardware:
+    cpus: 2
+    memory: 4Gi
+```
+A VM created with this VM class will have 2 vCPUs, 4 GiB of memory. Additionally, the VM will be created with a guaranteed 200MHz of CPU bandwidth and 1024 MB of memory. The CPU bandwidth of the VM will never be allowed to exceed 400MHz and the memory will not exceed 4Gi.
+
+!!! note "Units for CPU reservations/limits are in MHz"
+
+Please note that the units for CPU are different in hardware and reservations/limits.  Virtual hardware is specified in units of vCPUs, whereas CPU reservations/limits are in MHz.  Memory can be specified in MiB, GiB, whereas memory reservations/limits are always in MB.
 
 ### Networking
+Users can specify a list of network interfaces for their VM in the `spec.NetworkInterfaces` field. Each interface must specify the network type (one of `nsx-t`, or `vsphere-distributed` depending on the network provider being used). Users can further customize each interface with optional fields such as the Ethernet card, network name etc. For convenience, even if a VM does not specify any network interface, VM operator creates one connected to the default network available in the namespace. The IP address of the network interface is handled by the available network provider (configured by the administrator).
 
 ### Storage
+A VM deployed using VM operator inherits the storage defined in the `VirtualMachineImage`.  However, developers can also provision and manage additional storage dynamically by leveraging `PersistentVolumes`. To do this, a user would create a `PersistentVolumeClaim` resource by picking a `StorageClass` associated with their namespace, along with other properties such as the disk size, mode etc. VM operator then dynamically provisions a first class disks which is exposed to the guest as a block volume. Users can start using the disk after formatting and mounting it at a mountpoint. VM operator also supports resizing these volumes to increase their size.
```

Comment:
Do we wanna reference https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims here ?.

Reply:
Fair point.  Yeah, I will add that link there.